wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.        plt.gca().set_ylim(0.00, df['% of new articles about underrepresented regions'].max()) 
</pre><p>Is it necessary to override the default behavior here? I used to strict about always start the y axis at 0, but it seems like Wikicharts is following the guidance from the <a href="https://www.notion.so/wikimediadesign/Data-visualisation-guidelines-draft-b0c6246c28714bf384f3743e65d4b7b6" target="_blank">data visualization guidelines</a> to set the limits so that the line uses 2/3 of the full y range, starting above zero if necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>